code-studio/src/js/header.js

```diff
@@ -77,12 +77,17 @@ header.build = function (stageData, progressData, currentLevelId, scriptName, pu
   var isHeaderPopupVisible = false;
 
   function showHeaderPopup(target) {
-    sizeHeaderPopupToViewport();
-    $('.header_popup').show();
-    $('.header_popup_link_glyph').html('&#x25B2;');
-    $('.header_popup_link_text').text(dashboard.i18n.t('less'));
-    $(document).on('click', hideHeaderPopup);
-    lazyLoadPopup($(target).closest('.header_trophy_link').length > 0);
+    if ($(target).closest('.header_trophy_link').length > 0) {
+      // Only the 20-hour course has trophies.
+      location.href = '/s/20-hour#trophies';
```

Comment:
If the user clicks the trophy link in the header, we need to show them the trophies:

<img width="205" alt="trophies-link" src="https://cloud.githubusercontent.com/assets/413693/15590274/b248e398-234b-11e6-8e6f-8e28992cd108.png">

That used to happen by opening the progress dropdown and scrolling the page (but it's broken since PR https://github.com/code-dot-org/code-dot-org/pull/2292 because the progress dropdown resizes to fit the screen).

This PR changes the link to point to the `#trophies` anchor on the `/s/20-hour` page.